osu.Game/Graphics/UserInterfaceV2/DirectorySelector.cs

```diff
@@ -81,6 +81,8 @@ private void updateDisplay(ValueChangedEvent<DirectoryInfo> directory)
         {
             directoryFlow.Clear();
 
+            directory.NewValue?.Refresh();
```

Comment:
Rather than having this as a stand-alone call out here, why not move closer to the usage?